src/vcpkg/base/git.cpp

```diff
@@ -147,7 +147,7 @@ namespace vcpkg
 
         if (auto error = parser.get_error())
         {
-            return msg::format(msgGitUnexpectedCommandOutput).append_raw(error->format());
+            return msg::format(msgGitUnexpectedCommandOutput).append_raw(error->to_string());
```

Comment:
```suggestion
            return msg::format(msgGitUnexpectedCommandOutput).append_raw("\n").append_raw(error->to_string());
```
I think this needs a newline because `error->to_string()` expects to begin at start of line

Reply:
Also, it would be nice if `append_raw()` would eat anything that can be `Strings::append()`'d